po/R-zh_CN.po

```diff
@@ -391,100 +391,112 @@ msgid "i has evaluated to type"
 msgstr ""
 
 msgid ". Expecting logical, integer or double."
-msgstr ""
+msgstr "。期望布尔类型，整型或浮点型。"
 
 msgid "i evaluates to a logical vector length"
-msgstr ""
+msgstr "i 为一个布尔类型向量的长度。"
 
 msgid "but there are"
-msgstr ""
+msgstr "但是存在"
 
 msgid ""
 "rows. Recycling of logical i is no longer allowed as it hides more bugs than "
 "is worth the rare convenience. Explicitly use rep(...,length=.N) if you "
 "really need to recycle."
-msgstr ""
+msgstr "行。不再允许布尔值 i 循环补齐，尽管这样有些许便利，但会隐藏更多的 bug，得不偿失。
```

Comment:
each line needs to end with `"` and each line needs to begin with `"`

Reply:
Thanks for your review. I will fix the problems as soon as possible.